lib/core/providers/state/toolbox_state_provider.dart

```diff
@@ -64,6 +73,12 @@ class ToolBoxStateProvider extends _$ToolBoxStateProvider {
         state = state.copyWith(currentTool: ref.read(shapesToolProvider));
         ref.read(canvasPainterProvider.notifier).repaint();
         break;
+      case ToolType.SPRAY:
+        state = state.copyWith(currentTool: ref.read(sprayToolProvider));
+        final currentStrokeWidth = ref.read(paintProvider).strokeWidth;
+        (state.currentTool as SprayTool).updateSprayRadius(currentStrokeWidth);
+        ref.read(paintProvider.notifier).updateStrokeWidth(SPRAY_TOOL_RADIUS);
+        break;
```

Comment:
Little bit confusing.

Reply:
Sorry for the confusion. This is a "workaround", as the strokewidth would affect the density of the spraypattern, instead of the radius. This way we can "switch" the functionallity in order to alter the spray radius.